katas/content/nonlocal_games/ghz_quantum_strategy/Solution.qs

```diff
@@ -0,0 +1,27 @@
+namespace Kata {
+    operation AliceQuantum (bit : Bool, qubit : Qubit) : Bool {
+        if bit {
+            let res = MResetX(qubit);
+            return res == One;
+        }
+        let res = MResetZ(qubit);
+        return res == One;
+    }
+
+    operation BobQuantum (bit : Bool, qubit : Qubit) : Bool {
+        if bit {
+            let res = MResetX(qubit);
+            return res == One;
+        }
+        let res = MResetZ(qubit);
+        return res == One;
+    }
+
+    // alternative implementation
+    operation CharlieQuantum (bit : Bool, qubit : Qubit) : Bool {
+        if bit {
```

Comment:
Nice way to use the fact that there are three identical code fragments here! (No action required :-))